src/plugins/uploader.js

```diff
@@ -72,6 +72,13 @@ class Uploader {
 	}
 
 	static router(express) {
+		express.use(function (req, res, next) {
```

Comment:
This is bad, that registers a **global** handler for every single route path.
Meaning whatever route you register after that call will always fail if file uploads are disabled.

You have to scope this to the /uploads path